client/src/components/LibraryDashboard.js

```diff
@@ -0,0 +1,92 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/macro';
+
+export default function LibraryDashboard({
+  recentBooks,
+  onRenderBookDetails,
+  isStatic,
+}) {
+  return (
+    <LibraryCard isStatic={isStatic}>
+      <Link to="/mybooks">
+        <p>Recently added books</p>
+      </Link>
+      {recentBooks.length === 0 && (
+        <h5>
+          You have not added any books to your library yet. Start by searching
+          for a book.
+        </h5>
+      )}
+      {recentBooks.length > 0 &&
+        recentBooks.map((book) => (
+          <BookCard key={book.id} onClick={() => onRenderBookDetails(book)}>
+            <BookImage>
+              <img
+                src={book.volumeInfo?.imageLinks?.thumbnail}
+                alt="book cover"
+                width="36"
+                height="48"
+              />
+            </BookImage>
+            <BookTitle>
+              <p>{book.volumeInfo?.title}</p>
+            </BookTitle>
+            <AddedDate>
+              <p>Added on: {book.addToLibraryDate}</p>
+            </AddedDate>
+          </BookCard>
+        ))}
+    </LibraryCard>
+  );
+}
+
+const LibraryCard = styled.section`
+  background: var(--primary);
+  border-radius: var(--border-radius);
+  box-shadow: var(--box-shadow-offset-x) var(--box-shadow-offset-y)
+    var(--box-shadow-blur) var(--box-shadow-color);
+  margin: 0 auto;
+  min-height: ${(props) => (props.isStatic ? '' : '20vh')};
+  padding: 0.5rem;
+  width: ${(props) => (props.isStatic ? '375px' : '')};
+  a {
+    color: black;
+    text-decoration: none;
+  }
+  h5 {
+    text-align: center;
+  }
+`;
+
+const BookCard = styled.div`
+  align-items: center;
+  background: var(--background);
+  border-radius: var(--border-radius);
+  display: flex;
+  gap: 1rem;
+  justify-content: flex-start;
+  max-height: 64px;
+  margin: 0.5rem auto;
+  padding: 0.5rem;
+  width: 100%;
+`;
+
+const BookImage = styled.div``;
```

Comment:
do you need a styled component if there is no special styling? maybe consider simply using a div-Tag in the code